fs/share/man/man1/xpra.1

```diff
@@ -602,6 +602,10 @@ In \fIauto\fP mode, the client will set the bandwidth limit
 value to 80% of the maximum speed of the network interface
 it is using to connect to the server.
 
+\fB--splash\fP=\fIno\fP
+Do not display splash screen while connecting.
```

Comment:
Also, this is not just used for connecting, this is also shown when starting a server from a context which does have access to a GUI.